salt/orchestrate/edx/backup.sls

```diff
@@ -62,6 +63,24 @@ deploy_backup_instance_to_{{ ENVIRONMENT }}:
         - file: load_backup_host_cloud_profile
         - boto_iam_role: ensure_instance_profile_exists_for_backups
 
+{# Duplicity requires an archive directory otherwise it will have to create it and download files
+from s3 buckets when called. In order to accomodate that, we have an EBS volume that will be mounted
+by the ephemeral instance that is destroyed once backups are complete. #}
+{% set instance_id = salt.boto_ec2.find_instances('name=backups-{}'.format(ENVIRONMENT)) %}
+{% if instance_id %}
+attach_backup_volume:
+  salt.function:
+    - name: saltutil.runner
+    - arg:
+        - cloud.action
+    - kwarg:
+        func: ec2.attach_volume
+        kwargs:
+          instance_id: {{ instance_id }}
+          volume_name: {{ backup_volume_name }}
+          zone: us-east-1b
+          size: 400
```

Comment:
This is also not creating the volume... I think that what we want as an action instead is to use `ec2.create_attach_volume` which will allow us to specify the device name so that we don't have to attempt to find it via grains (which I don't believe will resolve properly anyway...).